src/common-util/src/main/java/ee/ria/xroad/common/util/CertUtils.java

```diff
@@ -136,6 +139,42 @@ private static String getPrincipalCommonName(X500Principal principal) {
         return cn;
     }
 
+    /**
+     * Reads subject alternative names from certificate and returns its string representation
+     * @param cert certificate for which to get the subject alternative names
+     * @return string representation of the subject alternative names
+     */
+    public static String getSubjectAlternativeNames(X509Certificate cert) {
+        List<String> fieldNames = Collections.unmodifiableList(
```

Comment:
This variable could be private static final constant in the CertUtils class, no need to recreate it every time this function is executed.